ansible/roles/accumulo/templates/log4j-service.properties

```diff
@@ -0,0 +1,24 @@
+log4j.appender.file=org.apache.log4j.RollingFileAppender
+log4j.appender.file.File=${accumulo.log.dir}/${accumulo.application}.log
+log4j.appender.file.MaxFileSize=1000MB
+log4j.appender.file.MaxBackupIndex=10
+log4j.appender.file.Threshold=ALL
+log4j.appender.file.layout=org.apache.log4j.PatternLayout
+log4j.appender.file.layout.ConversionPattern=%d{ISO8601} [%-8c{2}] %-5p: %m%n
+
+# send logs to Accumulo monitor
+log4j.appender.monitor=org.apache.accumulo.monitor.util.AccumuloMonitorAppender
+log4j.appender.monitor.Threshold=WARN
+
+# suppress spammy logging from 3rd party libs
+log4j.logger.org.mortbay.log=WARN
```

Comment:
Do we still need to suppress mortbay jetty stuff?

Reply:
I have no idea... I think I copied this from Accumulo 2.0

Reply:
I checked our dependency hierarchy and I don't think any mortbay stuff remains in Accumulo. It should probably be purged there also.

Reply:
Ok, I can remove that.